apps/passwd.c

```diff
@@ -369,9 +369,6 @@ static char *md5crypt(const char *passwd, const char *magic, const char *salt)
     if (magic_len > 0)
         salt_out += 2 + magic_len;
 
-    if (salt_len > 8)
-        goto err;
-
```

Comment:
This could be replaced with an assert() call.

Reply:
agreed.